service/cloudconfigv3/template.go

```diff
@@ -1,8 +1,48 @@
 package cloudconfigv3
 
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/randomkeys"
+)
+
 const (
 	set_ownership_etcd_data_dir_dropin = `[Unit]
 Requires=etc-kubernetes-data-etcd.mount
 After=etc-kubernetes-data-etcd.mount
+`
+	encryptionConfigTemplate = `
```

Comment:
What was the agreement with this template now? I understood it moved to k8scloudconfig. Why do we need it here still?

Reply:
its not move now, to align with aws-operator and k8s v2_0_0. It will be moved with k8s v3

Reply:
Is it already moved in v3 or is this yet to be done? If it is the latter one, who takes care about this? How do we make sure this is not forgotten?

Reply:
the PR is ongoing https://github.com/giantswarm/k8scloudconfig/pull/269
I would take care of this, in aws and kvm later

Reply:
Ok awesome.